isabelle/Doc/Specification/Specification.thy

```diff
@@ -11,54 +11,50 @@ section \<open>Introduction\<close>
-formalized with the proof-assistant Isabelle.
-\<close>
+With Marlowe we have a strong focus on operational semantics, as the goal was to develop a language that
+can also run on a blockchain \secref{sec:blockchain-agnostic}.\<close>
+
+text \<open>
+Smart contracts are immutable, that means that the assets deposited into a contract will inevitably follow
+the contract logic. In order to reduce the probability of not intended behaviour, the Marlowe DSL is designed
+with simplicity in mind. Without loops, recursion, or other features that general purposes smart-contract
+languages (E.g: Plutus, Solidity) have, it is easier to make certain claims. Each Marlowe contract can be
+reasoned with a static analyzer to avoid common pitfalls such as trying to Pay more money than
+available.\<close>
 
 text \<open>
 
 Chapter \secref{sec:marlowe-chaper} provides an overview of the Marlowe language. Chapter
 \secref{sec:marlowe-core} defines the Core language and semantics in detail. Chapter \secref{sec:marlowe-guarantees} presents
-proofs that guarantee that Marlowe contracts possess properties desirable for financial agreements.
+proofs of desirable properties for financial agreements.
 \<close>
 
 
 section \<open>The Marlowe Model\<close>
 
 text \<open>
-Marlowe \<^term>\<open>Contract\<close>s describe a series of steps, typically by describing the first step, together with
-another (sub-)contract that describes what to do next. For example, the contract
-@{term "Pay a p t v c"} says ``make a payment of @{term v} number of tokens @{term t} to the party
+Marlowe \<^term>\<open>Contract\<close>s describe a series of sequential steps. For example, the contract
+@{term "Pay a p t v c"} means ``make a payment of @{term v} number of tokens @{term t} to the party
 @{term p} from the account @{term a}, and then follow the contract @{term c}''. We call @{term c} the
-continuation of the contract.  All paths of the contract are made
-explicit this way, and each \<^term>\<open>Contract\<close> term is executed at most once.
-\<close>
+continuation of the contract. Strictly speaking the \<^term>\<open>Contract\<close> builds a tree structure with \<^term>\<open>Close\<close>
+\<^term>\<open>Contract\<close>s in the leafs and the branches represent all possible paths of a \<^term>\<open>Contract\<close>.\<close>
 
 subsection \<open>Data types\<close>
-text \<open>The \<^term>\<open>Value\<close>s and \<^term>\<open>Observation\<close>s \secref{sec:values-and-observations} only works with
+text \<open>The \<^term>\<open>Value\<close>s and \<^term>\<open>Observation\<close>s \secref{sec:values-and-observations} are restricted to 
 integers and booleans respectively. There is no native representation of decimal numbers, so in order to
 represent currencies it is recommended to work with fractional units (like cents). For example, amounts of
 Ada are represented in Lovelaces, because a Lovelace represents a one-millionth of an Ada. Dates are only
```

Comment:
The IOG style guide specifies that "ada" is lower case. However, I don't think we should grep through the document and try to change all occurrences.